<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-876IUID" name="factory.subsystems.monitoring.MonitoringSystemTest" tests="10" failures="0" timestamp="13 Jan 2019 11:27:13 GMT" time="0.422" errors="0">
  <testcase name="handleEvent" time="0.000" classname="factory.subsystems.monitoring.MonitoringSystemTest"/>
  <testcase name="handleError" time="0.038" classname="factory.subsystems.monitoring.MonitoringSystemTest"/>
  <testcase name="handleGlobalError" time="0.372" classname="factory.subsystems.monitoring.MonitoringSystemTest"/>
  <testcase name="addOrder" time="0.004" classname="factory.subsystems.monitoring.MonitoringSystemTest"/>
  <testcase name="handleEventCheckThatErrorEventHandlerIsNotUsed" time="0.003" classname="factory.subsystems.monitoring.MonitoringSystemTest"/>
  <testcase name="start" time="0.001" classname="factory.subsystems.monitoring.MonitoringSystemTest"/>
  <testcase name="handleEventHandlingException" time="0.001" classname="factory.subsystems.monitoring.MonitoringSystemTest"/>
  <testcase name="handleNormalEvent" time="0.000" classname="factory.subsystems.monitoring.MonitoringSystemTest"/>
  <testcase name="stop" time="0.001" classname="factory.subsystems.monitoring.MonitoringSystemTest"/>
  <testcase name="handleNewOrder" time="0.002" classname="factory.subsystems.monitoring.MonitoringSystemTest"/>
</testsuite> <!-- factory.subsystems.monitoring.MonitoringSystemTest -->
